can_tools/scrapers/official/ME/__init__.py

```diff
@@ -0,0 +1 @@
+from .me_vaccines import MaineCountyVaccines
```

Comment:
We've used global imports everywhere, going to change now

Reply:
Ha! Ooops. Thanks for catching this.